src/app/helpers/environmentVariablesHelper.js

```diff
@@ -2,277 +2,56 @@
 const env = process.env
 const fs = require('fs')
 const packageObj = JSON.parse(fs.readFileSync('package.json', 'utf8'))
-const SB_DOMAIN = 'https://staging.open-sunbird.org'
```

Comment:
You should not remove all the other env properties.. 
I would suggest you to split the files into 2 
Mandatory & optional. 

Merge the while building the application..